tests/x11/yast2_lan_restart.pm

```diff
@@ -149,68 +168,71 @@ sub del_device {
+    send_key 'alt-y';                     # Dynamic address
+    send_key 'alt-n';                     # Next
     close_yast;
 }
 
 sub test_2 {
     diag '__________(2) Start yast2 lan -> Edit (a NIC) -> no change, don\'t switch to another tab, [Next] -> [OK]__________';
     run_yast2_lan_edit;
-    send_key 'alt-n';                                                 # Next
+    send_key 'alt-n';                     # Next
     check_network;
 }
 
 sub test_5 {
     diag '__________(5) Start yast2 lan -> Edit (a NIC) -> no change, switch to Hardware tab, [Next] -> [OK]__________';
     run_yast2_lan_edit;
-    send_key 'alt-w';                                                 # Hardware tab
+    send_key 'alt-w';                     # Hardware tab
     assert_screen 'yast2_lan_hardware_tab';
-    send_key 'alt-n';                                                 # Next
+    send_key 'alt-n';                     # Next
     check_network;
 }
 
 sub test_6 {
     diag '__________(6) Start yast2 lan -> Edit (a NIC) -> no change, switch to General tab, [Next] -> [OK]__________';
     run_yast2_lan_edit;
-    send_key 'alt-g';                                                 # General tab
+    send_key 'alt-g';                     # General tab
     assert_screen 'yast2_lan_general_tab';
-    send_key 'alt-n';                                                 # Next
+    send_key 'alt-n';                     # Next
     check_network;
 }
 
 sub test_7 {
     my $dev_name = shift;
     diag '__________(7) Start yast2 lan -> Edit (a NIC) -> switch to Hardware tab, change nic name, [Next] -> [OK] -> device name is changed__________';
     run_yast2_lan_edit;
-    send_key 'alt-w';                                                 # Hardware tab
+    send_key 'alt-w';                     # Hardware tab
     assert_screen 'yast2_lan_hardware_tab';
-    send_key 'alt-e';                                                 # Change device name
+    send_key 'alt-e';                     # Change device name
     assert_screen 'yast2_lan_device_name';
-    send_key 'alt-m';                                                 # Udev rule based on MAC
+    send_key 'alt-m';                     # Udev rule based on MAC
     send_key 'tab';
     send_key 'tab';
     send_key 'tab';
     send_key 'tab';
     type_string "$dev_name";
-    send_key 'alt-o';                                                 # OK
-    send_key 'alt-n';                                                 # Next
+    send_key 'alt-o';                     # OK
+    send_key 'alt-n';                     # Next
     check_network('restart');
 }
 
 sub run {
     select_console 'x11';
     x11_start_program("xterm -geometry 155x50+5+5", target_match => 'xterm');
     become_root;
+    # make sure that firewalld is stopped, or we have later pops for firewall activation warning
+    # or timeout for command 'ip a' later
+    script_run "systemctl stop firewalld";
```

Comment:
Please don't add a `script_run` without checking the return value.

Reply:
It was assert_script_run, then I change it into script_run. But it is fine with assert_script_run. Please check, thanks!